### PR TITLE
Update kubecon-eu-2023.yaml

### PR DESCRIPTION
**Pull request by pnawars**

### CODE REVIEW

File: conferences/kubecon-eu-2023.yaml [hunk 1]
```diff
@@ -163,3 +163,13 @@ parties:
     ​🎳 THE KUBERBOWL! 🎳
   notes: |
     Requires registering online
+- name: Amsterdam Open- Source Data Infrastructure
+  date: 2023-04-19
+  time: 6:30 PM - 9:00 PM
```
Comment (dveeden) [inline conferences/kubecon-eu-2023.yaml]:
```suggestion
  time: "6:30pm - 9pm"
```

File: conferences/kubecon-eu-2023.yaml [hunk 2]
```diff
@@ -163,3 +163,13 @@ parties:
...
+  website: https://www.meetup.com/amsterdam-open-source-data-infrastructure-meetup/events/292274551/
+  location: Spaces Herengracht - The ProjectRoom, Herengracht 124, 1015 BT Amsterdam,Netherlands
+  description: Are you interested in learning more about open-source data technologies?
```
Comment (dveeden) [inline conferences/kubecon-eu-2023.yaml]:
```suggestion
  description: |
    Are you interested in learning more about open-source data technologies?
  notes: |
    Requires registering online
```

**Comment by dveeden** [comment]
I guess @AverageMarcus is the person to review/approve this?

**Comment by AverageMarcus** [comment]
Amazing! Thanks!

**Pull request merged**